app/assets/stylesheets/steps.scss

```diff
@@ -88,7 +89,6 @@
           display: flex;
           flex-shrink: 1;
           height: 39px;
-          min-width: 100px;
```

Comment:
Why you remove it? Without `min-width' step name will be covered with big author name.